tests/test_environ.py

```diff
@@ -108,22 +109,25 @@ def p():
         return '{user}'
     assert isinstance(partial_format_prompt(p), str)
 
-def test_HISTCONTROL():
+def test_HISTCONTROL_none():
```

Comment:
PEP8 functions names should be lowercase with underscores.  I know that you didn't write it this way originally @laerus, but if you don't mind cleaning it up since you touched these lines, that would be awesome! Thanks a ton.